### ISSUE TITLE
Missing location when making the call is URL is not ended by '/'

### ISSUE THREAD

**Issue by ThomasdOtreppe** [logstash-plugins/logstash-output-http#14]
Here is my configuration for the plugin

```
http {
url => "http://10.0.0.1:8080"
http_method => "post"
}
```

When it does the post, it looks like this:
```
POST  HTTP/1.1
```

You can see the location is clearly missing. Based on what I see, just ending the URL with '/' will fix it. 

In order to fix it, assume '/' when that part is empty.

**Comment by jsvd** [comment]
This is due to https://github.com/jordansissel/ruby-ftw/issues/29

**Comment by andrewvc** [comment]
@ThomasdOtreppe this should be fixed in the latest version (1.1.0) which no longer uses FTW. If this issue persists please reopen this issue or open a new one.
